app/api/orders.py

```diff
@@ -68,14 +68,16 @@ def is_payment_valid(order, mode):
         return (order.paid_via == 'paypal') and order.transaction_id
 
 
-def check_billing_info(data):
-    if data.get('amount') and data.get('amount') > 0 and not data.get('is_billing_enabled'):
+def is_billing_info(data, order):
```

Comment:
`is_billing_info`?

Reply:
If `is_billing_info` needed for all orders or not.